.github/workflows/sauce-labs.yaml

```diff
@@ -58,7 +60,7 @@ jobs:
           if [ "$BROWSER_NAME" = "firefox" ]; then EXCLUDE_GROUP+="exclude-firefox,"; fi
           if [ "$BROWSER_NAME" = "chrome" ]; then EXCLUDE_GROUP+="exclude-chrome,"; fi
           if [ -n "$EXCLUDE_GROUP" ]; then EXTRA_PARAMS+=" --exclude-group $EXCLUDE_GROUP"; fi
-          ./vendor/bin/phpunit --coverage-clover ./logs/coverage-clover.xml $EXTRA_PARAMS
+          ./vendor/bin/phpunit --testsuite functional --coverage-clover ./logs/coverage-clover.xml $EXTRA_PARAMS
```

Comment:
I will not recommend using `composer run test` because the composer timeout would break the running command